src/main/kotlin/no/nav/tilleggsstonader/sak/vedtak/boutgifter/dto/BeregningsresultatBoutgifterDto.kt

```diff
@@ -11,14 +11,15 @@ import java.time.LocalDate
 
 data class BeregningsresultatBoutgifterDto(
     val perioder: List<BeregningsresultatForPeriodeDto>,
+    val skalBrukeDetaljertVisning: Boolean,
```

Comment:
Burde feltet være at periodene inneholder utgifterOvernatting, så kan frontend selv styre om det skal vise detaljer eller ikke? 

```suggestion
    val inneholderUtgifterOvernatting: Boolean,
```

Reply:
Ja - det var lurt.
https://github.com/navikt/tilleggsstonader-sak/pull/709/commits/bb1c649e943a85a3c876311a5c64ffb85b8630ac